tests/check_backdated_set_cost.rs

```diff
@@ -0,0 +1,89 @@
+mod test_init;
+
+use chrono::{DateTime, Utc};
+use json::{array, object};
+use serde_json::from_str;
+use std::io;
+use std::str::FromStr;
+use std::sync::Arc;
+use test_init::init;
+use uuid::Uuid;
+
+use crate::test_init::{create_record, goods, receive, store, transfer};
+use nae_backend::commutator::Application;
+use nae_backend::memories::MemoriesInFiles;
+use nae_backend::storage::Workspaces;
+use service::utils::json::JsonParams;
+use service::{Context, Services};
+use store::balance::{BalanceForGoods, Cost};
+use store::batch::Batch;
+use store::elements::{dt, Goods, Mode, Qty, Store};
+use store::operations::{InternalOperation, OpMutation};
+use store::process_records::process_record;
+use store::GetWarehouse;
+
+const WID: &str = "yjmgJUmDo_kn9uxVi8s9Mj9mgGRJISxRt63wT46NyTQ";
+
+#[actix_web::test]
+async fn check_backdated_set_cost() {
```

Comment:
what kind of new checks here vs `check_zero_batch_deleted`?

Reply:
same